content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -120,6 +120,172 @@ see the [API reference](/docs/reference/kubernetes-api/) for more information. I
 is not possible to access sub-resources across multiple resources - generally a new
 virtual resource type would be used if that becomes necessary.
 
+## HTTP media types {#alternate-representations-of-resources}
+
+Over HTTP, Kubernetes supports JSON and Protobut wire encodings.
+
+{{% note %}}
+Although YAML is widely used to define Kubernetes manifests locally, Kubernetes does not
+support the [`application/yaml`](https://www.rfc-editor.org/rfc/rfc9512.html) media type
+for API operations.
```

Comment:
There is `application/apply-patch+yaml` (which is, uh, JSON). For simplicity I am deliberately not mentioning that detail here. The statement:
```markdown
Kubernetes does not support the [`application/yaml`](https://www.rfc-editor.org/rfc/rfc9512.html)
media type for API operations.
```
remains true.

Reply:
+100, this is an implementation detail that, in context of this doc, doesn't seem helpful.

Reply:
I don't think this is correct. It contradicts with my previous understanding. So I tried this:

```
SERVER=https://192.168.1.201:6443
curl -k --key ~/tmp/k8s.key --cert ~/tmp/k8s.crt \
  -X POST \
  -H "Content-Type: application/yaml" \
  --data-binary @cm1.yaml \
  ${SERVER}/api/v1/namespaces/default/configmaps
```

Then I did get the ConfigMap created.
This means 'application/yaml', as a content-type, is supported.

Reply:
The apiserver definitely supports receiving yaml in API requests

Reply:
And returning it

Reply:
OK, bug issues are welcome!

I wonder when we added that.

Reply:
(when I started using Kubernetes, `application/yaml` wasn't a registered media type, although some people used it anyway)

Reply:
See https://github.com/kubernetes/website/pull/46230#issuecomment-2408947763

Reply:
@tengqm, thanks for pointing this out.